src/main/java/core/basesyntax/Calculator.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class Calculator {
+
+    public double calculate(double a, double b, char operator) {
+        switch (operator) {
+            case '+': {
+                return a + b;
```

Comment:
```suggestion
            case '+':
                return a + b;
```
we don't need curly braces for each 'case' clause